### ISSUE TITLE
Linebreak after ~35 characters 

### ISSUE THREAD

**Issue by sierrax369** [zerowriter/zerowriter1#22]
In current version, the zerowriter1 (in my case waveshare_2.2 branch) is saving the lines after hitting return, in the same way it showed on the 35x13 lines framed display.
This makes it a little difficult to edit the text afterwards.  
Old typewriters should have around 60 types per line. Old Computer around 80. On modern computers, typists write entire paragraphs without using the return button.
Wouldn't it make sense to save the stuff 2 times? One time in the Displayversion and one time in the raw typed version?

**Comment by sierrax369** [comment]
Another way could be to use 2 returns for paragraphs. And 'sed' then the the files… but this is another step and you have to be strict.

**Comment by zerowriter** [comment]
yes! the software should be updated to output more cleanly in respect to formatting. It's on my to-do list.

**Comment by digitalsucks** [comment]
Would love this update. Been experimenting with my DIY build for a few weeks, and this is the one workflow snag that keeps bugging me. So, just putting in my vote! It's a really great writer.

**Comment by zerowriter** [comment]
This is certainly at the top of my list to change when I dig in to it - I
coded this in a very crude way.
